packages/types/src/deprecated.ts

```diff
@@ -0,0 +1,45 @@
+export type ModeOption = {
```

Comment:
It looks like I added them to this new deprecated file, but I never actually removed them from the old main types file. I am wondering if it would be better to simply leave these as is. They don't actually conflict with any of the new stuff.